sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Azure.MixedReality.ObjectAnchors.Conversion.csproj

```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="Azure.MixedReality.Authentication" VersionOverride="1.0.1" />
+    <PackageReference Include="Azure.MixedReality.Authentication" VersionOverride="1.1.0" />
```

Comment:
Why do we need to override?  Does something prevent us from bumping the central package version?

Reply:
There are repo policies that require you either set the version centrally or you have to override. This library is consumed by a few other Azure SDKs and I imagine this was the solution so that each could update independently.

Reply:
We very strongly discourage overrides outside of specific temporary needs.   To ensure that the packages in the Azure SDK work together as seamlessly as possible, we want to manage versions centrally and have packages use them.